### PR TITLE
ENH: Add s rep extension

### PR DESCRIPTION
**Pull request by ZhiyLiu**
Please help to add this extension descriptor.

### CODE REVIEW

File: SkeletalRepresentation.s4ext [hunk 1]
```diff
@@ -0,0 +1,44 @@
+#
+# First token of each non-comment line is the keyword and the rest of the line
+# (including spaces) is the value.
+# - the value can be blank
+#
+
+# This is source code manager (i.e. svn)
+scm git
+scmurl https://github.com/ZhiyLiu/SlicerSkeletalRepresentation.git
```
Comment (pieper) [inline SkeletalRepresentation.s4ext]: Wouldn't this be hosted out of the KitwareMedical repository?

File: SkeletalRepresentation.s4ext [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+# - These should be names of other modules that have .s4ext files
+# - The dependencies will be built first
+depends NA
+
+# Inner build directory (default is ".")
+build_subdirectory .
+
+# homepage
+homepage http://slicer.org/slicerWiki/index.php/Documentation/Nightly/Extensions/SkeletalRepresentation
+
```
Comment (pieper) [inline SkeletalRepresentation.s4ext]: The wiki page is empty, maybe point this to the README.md instead (and add a wiki stub that points to the readme).

File: SkeletalRepresentation.s4ext [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+# Firstname1 Lastname1 ([SubOrg1, ]Org1), Firstname2 Lastname2 ([SubOrg2, ]Org2)
+# For example: Jane Roe (Superware), John Doe (Lab1, Nowhere), Joe Bloggs (Noware)
+contributors Zhiyuan Liu (University of North Carolina at Chapel Hill), Junpyo Hong (University of North Carolina at Chapel Hill), Stephen M. Pizer (University of North Carolina at Chapel Hill), Jared Vicory (Kitware, Inc.), Pablo Hernandez-Cerdan (Kitware, Inc.), Beatriz Paniagua (Kitware, Inc.), Jean-Christophe Fillion-Robin (Kitware, Inc.)
+
+# Match category in the xml description of the module (where it shows up in Modules menu)
+category Topology, Skeletonization
+
+# url to icon (png, size 128x128 pixels)
+iconurl https://github.com/KitwareMedical/SlicerSkeletalRepresentation/blob/master/SkeletalRepresentation.png
```
Comment (pieper) [inline SkeletalRepresentation.s4ext]: Nice icon!

File: SkeletalRepresentation.s4ext [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+# Give people an idea what to expect from this code
+#  - Is it just a test or something you stand behind?
+status 
+
+# One line stating what the module does
+description Toolkit that provides users to establish, refine, and visualize skeletal representation of an object.
+
+# Space separated list of urls
+screenshoturls https://github.com/KitwareMedical/SlicerSkeletalRepresentation/blob/master/VisualizationScreenshot.png
```
Comment (pieper) [inline SkeletalRepresentation.s4ext]: I believe this should be: https://cdn.rawgit.com/KitwareMedical/SlicerSkeletalRepresentation/529b95bc/VisualizationScreenshot.png

**Comment by jcfr** [comment]
Hi,

I suggest to change the repo to
https://github.com/KitwareMedical/SlicerSkeletalRepresentation

What do you think Bea ?

Thanks
Jc

**Comment by ZhiyLiu** [comment]
I fix it [here](https://github.com/Slicer/ExtensionsIndex/pull/1581/commits/781d2675f167bfe296ad1ab3237a6fe2e3d8504e), and close this request to new a request

**Pull request closed without merging**